app/src/main/java/com/example/tungcon/demo/movie/IamgeMovie/ItemMovieActivity.java

```diff
@@ -0,0 +1,48 @@
+package com.example.tungcon.demo.movie.IamgeMovie;
+
+import android.content.Context;
+import android.content.Intent;
+import android.os.Bundle;
+import android.support.v7.app.AppCompatActivity;
+import android.widget.ImageView;
+import android.widget.TextView;
+
+import com.bumptech.glide.Glide;
+import com.example.tungcon.demo.R;
+
+public class ItemMovieActivity extends AppCompatActivity {
+    private static final String EXTRA_IMAGE = "iamge";
+    private static final String EXTRA_NAMEMOVIE = "namemovie";
+    private ImageView mImageMovie;
+    private TextView mTextViewMovie;
+
+    public static Intent getDataIntent(Context context, String movie, String imageMovie) {
```

Comment:
em nên truyền cả đối tượng MOvie sang
```suggestion
    public static Intent getDataIntent(Context context, Movie movie) {
```